src/Hedgehog/Gen.fs

```diff
@@ -416,27 +422,28 @@ module Gen =
 
     /// Generates a random 64-bit floating point number.
     let double (range : Range<double>) : Gen<double> =
-        create (Shrink.towardsDouble <| Range.origin range) (Random.double range)
+        Random.double range
+        |> create (Shrink.towardsDouble (Range.origin range))
 
     /// Generates a random 64-bit floating point number.
     let float (range : Range<float>) : Gen<float> =
-        (double range) |> map float
+        double range |> map float
 
     /// Generates a random 32-bit floating point number.
     let single (range : Range<single>) : Gen<single> =
-      double (Range.map ExtraTopLevelOperators.double range) |> map single
+        double (Range.map ExtraTopLevelOperators.double range) |> map single
```

Comment:
Good catch :+1:

Reply:
I think that was the newly added `.editorconfig` file.  There were some other lines changed because white space was removed from the end of the line.